markdown/chapters/meandeck/basics.md

```diff
@@ -145,4 +145,9 @@ of Will available.
 If you want to discover a bit by yourself before diving into more detailed
 chapters, try a couple goldfish games with the following proven list.
 
-<deck path="meandeck.ubrg.txt" />
+<deck path="meandeck.ubr.txt" />
+
+Alternatively, if you're looking into getting the cards here is a functional
+budget version:
+
+<deck path="meandeck.budget.txt" />
```

Comment:
Obvious upgrades as budget allows:

#### Access to {{Force of Will}}:

```
-2 Flusterstorm
-2 Thoughtseize
+4 Force of Will
```

#### Access to 1+ {{Undergound Sea}}:

```
-1 Watery Grave
+1 Underground Sea
```

At 2 {{Underground Sea}}, {{Daze}} is unlocked as a reliable option.

Reply:
Added in the 15th FAQ :pray: